packages/types/src/network.ts

```diff
@@ -0,0 +1,162 @@
+import { Blockchains } from './blockchain';
+import {
+  BITCOIN_API_BASE_URL_MAINNET,
+  BITCOIN_API_BASE_URL_SIGNET,
+  BITCOIN_API_BASE_URL_TESTNET,
+  HIRO_API_BASE_URL_MAINNET,
+  HIRO_API_BASE_URL_TESTNET,
+} from './constants';
```

Comment:
I'm not sure types should import from constants.

Can't we put only the type values in here, and then import them in constants with the strings? So we export the networks from there?

Reply:
Trying to understand this comment??? 🤔

Reply:
See latest diagram, but this puts constant values in our types package, which I think we should avoid